etc/examples/caffe-gpu-model/lenet_solver.prototxt

```diff
@@ -0,0 +1,25 @@
+# The train/test net protocol buffer definition
```

Comment:
Do we need to duplicate this whole file for a minimal change? Can we ask users to change 1/2 params manually - or give a script?
https://github.com/IBM/FfDL/blob/master/etc/examples/caffe-model/lenet_solver.prototxt